app/src/main/java/org/simple/clinic/teleconsultlog/drugduration/DrugDurationInit.kt

```diff
@@ -0,0 +1,17 @@
+package org.simple.clinic.teleconsultlog.drugduration
+
+import com.spotify.mobius.First
+import com.spotify.mobius.First.first
+import com.spotify.mobius.Init
+
+class DrugDurationInit : Init<DrugDurationModel, DrugDurationEffect> {
+
+  override fun init(model: DrugDurationModel): First<DrugDurationModel, DrugDurationEffect> {
+    val effects = mutableSetOf<DrugDurationEffect>()
+    if (model.hasDuration) {
+      effects.add(SetDrugDuration(model.duration))
```

Comment:
Since the view is an edit text, do we need to do this since it will persist its state correctly?

Reply:
It was mainly for setting the default duration that is passed from the screen arguments. Should I have used a different approach for that?

Reply:
Ah, I see. This makes sense.